experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -115,7 +115,7 @@ function update_field!(sim::BucketSimulation, ::Val{:air_density}, field)
     parent(sim.integrator.p.bucket.ρ_sfc) .= parent(field)
 end
 
-step!(sim::BucketSimulation, t) = step!(sim.integrator, t - sim.integrator.t, true)
+step!(sim::BucketSimulation, t) = t - sim.integrator.t > 0 ? step!(sim.integrator, t - sim.integrator.t, true) : nothing
```

Comment:
Hmm, I'm seeing this issue with updating the slabs as well. Maybe it would be cleaner to add `reinit!` to the driver? Testing this rn.

Reply:
hmmm does it happen with atmos too? not sure why we are seeing it with CTS for the slabs and the bucket but not clima atmos.

Reply:
In the current driver we were already reinitialize atmos, i.e. `reinit!(atmos_sim)` (to set nonzero radiative fluxes). `reinit_model_sims!(cs.model_sims)` works locally btw: https://github.com/CliMA/ClimaCoupler.jl/pull/387/files

Reply:
Otherwise LGTM! ☺️

Reply:
yay! 

I will try the same change here, nice solution!

Reply:
it works btw!